RELEASE.md

```diff
@@ -1,14 +1,57 @@
-# *Infrastructure - Dependency Updates*
+# *Infrastructure - CDM Artifacts published to Maven Central*
 
-_What is being released?_
+#### _What is being released?_
 
-This release updates the `rosetta-dsl` dependency:
+As part of the CDM migration to FINOS, all generated artifacts are now being published to Maven Central. The intention to make this change was presented to CDM community, and noted in the CDM transition document ["Common Domain Model becoming Open Source @ FINOS"](https://assets.isda.org/media/6fac704a/081f5537.pdf), earlier this year. This is a positive step for the CDM community and realizes a demand from implementors that the project has noted for some time.
 
-- Versions:
-    - `7.5.4` Fixed DSL issue related to `typeAlias` - see https://github.com/REGnosys/rosetta-dsl/issues/554 
+#### _Downloading Source Code_
 
-This release contains no changes to the model or test expectations.
+Precviously, only the latest CDM source code was available to download from the CDM Portal. After this release, the open source code is only available to be downloaded from [CDM GitHub](https://github.com/finos/common-domain-model). However, additional to the latest version, previously released versions are tagged and can be downloaded from [CDM GitHub Releases](https://github.com/finos/common-domain-model/releases) also.
 
-_Review directions_
+#### _CDM as Java_
 
-CDM Java implementors should update their maven `pom.xml` to the latest CDM maven artefact (groupId com.isda, artifactId cdm) and recompile.
+In order to use the latest version of CDM in a [Maven](https://maven.apache.org) project, the following dependency needs to be added to the project pom.xml:
+
+```
+<dependency>
+  <groupId>org.finos.cdm</groupId>
+  <artifactId>cdm-java</artifactId>
+  <version>LATEST</version>
+</dependency>
+```
+
+Going forward from this release, the "LATEST" value can be substituted for with any of the subqequent [CDM GitHub Releases](https://github.com/finos/common-domain-model/releases). 
+
+For implemntation of CDM releases prior to today's release, dependencies in implementers' projects can still point to the version in [Artifactory](https://regnosys.jfrog.io/ui/repos/tree/General/libs-snapshot/com/isda/cdm) as they do today.
```

Comment:
Typo plus above you use implementor and here you use implementer... think they are both valid but better to stick to one spelling